<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Close</class>
 <widget class="QDialog" name="Close">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Closing the Logger</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="notice">
     <property name="text">
      <string>&lt;b&gt;WARNING!&lt;/b&gt; Closing the logger will cause any clients connected through the logger to be disconnected from the server. You should ensure all client applications are closed before doing this.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="ignoreCheckbox">
     <property name="text">
      <string>Fuck off! Don't bother me about this again.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="yesButton">
       <property name="text">
        <string>Fuck Em!</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="noButton">
       <property name="text">
        <string>Save My n00b Ass!!!</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>noButton</tabstop>
  <tabstop>ignoreCheckbox</tabstop>
  <tabstop>yesButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
